### ISSUE TITLE
Continuation of the project

### ISSUE THREAD

**Issue by savchenko** [Drive-Trust-Alliance/sedutil#369]
Seeing @r0m30 pushing new changes after a short (2017-2021) break is exciting. Is there a plan to continue development? There is currently [161 fork](https://github.com/Drive-Trust-Alliance/sedutil/network/members) on GitHub only. Is there a plan to pull changes from @amotin, @ChubbyAnt, @ladar and others?

**Comment by r0m30** [comment]
Plan? COVID has taught me that plans are tenuous at best. What I hope to accomplish is the following:
Return the Windows version to feature parity (NVMe support). The code is ready but we will have to drop support for pre-windows 20H1 because of the way windows implemented NVMe support.￼
Target the PBA images to more recent kernels.
Improve the compatibility of the PBA images.
Bug fixes.￼￼

I also plan to drop all 32 bit support.
￼
I do not plan to scour the forks for features/improvements. We need to protect our copyright so that we can re-license the code￼ in ways that allow monetization. 
￼

**Comment by savchenko** [comment]
> Return the Windows version to feature parity (NVMe support). The code is ready but we will have to drop support for pre-windows 20H1 because of the way windows implemented NVMe support

Brave. Do you have any sort of understanding with Microsoft on this? In other words, will it be of much use given the upcoming Windows 11, change in API/ABI, etc.

> Target the PBA images to more recent kernels.
> Improve the compatibility of the PBA images.
> Bug fixes.

❤

> I also plan to drop all 32 bit support.

Makes sense.

> I do not plan to scour the forks for features/improvements. We need to protect our copyright so that we can re-license the code in ways that allow monetization.

Understandable. What if people send their changes back to the upstream? I think this is more likely if `LICENSE` is added to the repository, currently there is only the 5 pages long `.docx` document.

Also, speaking of monetisation, enabling some sort of donations interface might be a start.

**Comment by r0m30** [comment]
The NVMe support in windows uses documented APIs only so I don't see any issues going forward.
￼
There is a contributing file in the codebase. I do understand why people don't want sign the agreement especially for trivial updates but not a single PR includes the contributor agreement. 

We do have a patrion link in the readme.md, is it not prominent enough?

**Comment by savchenko** [comment]
> We do have a patrion link in the readme.md, is it not prominent enough?

Not at all. You might want to enable sponsorship in repository settings and point it to your Patreon.

>  There is a contributing file in the codebase. I do understand why people don't want sign the agreement especially for trivial updates but not a single PR includes the contributor agreement.

Current agreement looks like "I need to pay a lawyer to review it" which leads to dilution of the effort, multiplication of forks and so on. Any chance it can be shortened and simplified? Then people who have the most mature forks can be contacted and asked if they are willing to contribute back.
